### ISSUE TITLE
Suggestion: Worktrains

### ISSUE THREAD

**Issue by TheFrommie86** [trolley813/dv-rolling-stock-numbering#3]
CAB for Caboose
HC1 for Handcar
BE2 for Microshunter
DES for the Slug

**Comment by trolley813** [comment]
Yes, the update is already on the way. Thanks!

**Comment by trolley813** [comment]
Fixed in [1.1.0](https://github.com/trolley813/dv-rolling-stock-numbering/releases/tag/v1.1.0), although the prefixes are slightly different.

**Comment by TheFrommie86** [comment]
Perhaps it would be better making "DE6S" the prefix for the Slug. To avoid Confusuin with the upcoming E6 Locomotive

**Comment by TheFrommie86** [comment]
> Fixed in [1.1.0](https://github.com/trolley813/dv-rolling-stock-numbering/releases/tag/v1.1.0), although the prefixes are slightly different.

Perhaps it would be better making "DE6S" the prefix for the Slug. To avoid confusion with the upcoming E6 Locomotive

**Comment by TheFrommie86** [comment]
Will there be an Update for the Utility Railbus?
URV Perhaps

**Comment by trolley813** [comment]
Yes, it will. Probably "DM1" (for both the DM1U and the yet-to-be-added passenger DM1P, since the former was converted from the latter)

**Comment by trolley813** [comment]
Added in version 1.2.0
